cluster/points_writer.go

```diff
@@ -233,6 +237,9 @@ func (w *PointsWriter) WritePoints(p *WritePointsRequest) error {
 		}(shardMappings.Shards[shardID], p.Database, p.RetentionPolicy, points)
 	}
 
+	// Fire and forget points to Subscriber
+	go w.Subscriber.WritePoints(p)
```

Comment:
Is there going to be bad overhead with launching  a new goroutine on every write? Just curious if this is necessary since the writer in question in UDP. I would leave it to the `Subscriber` to decide in its `WritePoints` implementation if it should be kicked off in a goroutine.